src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -12,7 +12,7 @@ public static class MsalError
     {
         /// <summary>
         /// Standard OAuth2 protocol error code. It indicates that the application needs to expose the UI to the user
-        /// so that the user does an interactive action in order to get a new token.
+        /// so that the user does an interactive action in order to get a new token. Or you may be using multiple toke cache for the same user.
```

Comment:
nit: not sure about the additional message here. i don't think it's necessary and not a common issue. there shouldn't be multiple caches for the same user.

Reply:
Removed.